cog/daily_charge.py

```
@@ -23,95 +25,293 @@
         )
-        await interaction.response.send_message(embed=self.embed, ephemeral=True)
 
-    async def channel_error(self, interaction):
-        self.embed = discord.Embed(color=0xFF0000)
-        self.embed.set_thumbnail(url="https://http.cat/images/404.jpg")
-        self.embed.add_field(name="這裡似乎沒有打雷…", value="  ⛱️", inline=False)
-        self.embed.add_field(name="到「每日充電」頻道試試吧！", value="", inline=False)
-        # 其他文案：這裡似乎離無線充電座太遠了，到「每日充電」頻道試試吧！ 待商議
-        await interaction.response.send_message(embed=self.embed, ephemeral=True)
+    @staticmethod
+    def is_cross_day(last_charge: datetime, executed_at: datetime):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `is_cross_day`

(ANN205)